spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/ColumnRef.scala

```diff
@@ -17,6 +17,9 @@ sealed trait NamedColumnRef extends ColumnRef {
   /** Returns a name of the selection as it is seen in the result set. Most likely this is going to be
     * used when providing custom column name to field name mapping. */
   def selectedAs: String
+
+  /** Returns an alias for the referenced column */
+  def alias: Option[String]
```

Comment:
I don't like the fact it is an `Option`. It doesn't tell me, what to do if it is not set. Maybe it should be just a `String` set to `selectedAs` if alias was not given?